Modules/unicodedata.c

```diff
@@ -92,7 +92,6 @@ static PyMemberDef DB_members[] = {
 
 /* forward declaration */
 static PyTypeObject UCD_Type;
-#define UCD_Check(o) Py_IS_TYPE(o, &UCD_Type)
```

Comment:
I would prefer to keep UCD_Check() macro, and move "self &&" test inside the macro. Maybe rename "o" to "self", and explain that self is NULL when the PyCapsule API is used.